libraries/chain/wasm_interface.cpp

```diff
@@ -1030,17 +1030,17 @@ class memory_api : public context_aware_api {
          return (char *)::memset( dest, value, length );
       }
 
-      uint32_t sbrk(int num_bytes) {
+      int sbrk(int num_bytes) {
```

Comment:
typo: the return variable `prev_num_bytes` is [uint32_t](https://github.com/EOSIO/eos/blob/709a68eaf7bf9f0c872ee94f20048fa131c911e5/libraries/chain/wasm_interface.cpp#L1047)

Reply:
I think the return type of `sbrk` should still be `uint32_t`

Reply:
I changed it to int to be a more clear indicator that `sbrk()` can now return a negative value. While I realize it could stay `uint32_t` and still return -1 or even 0xFFFFFFFF that doesn't seem as explicit.